saleor/graphql/meta/tests/test_meta_mutations.py

```diff
@@ -87,7 +88,7 @@ def execute_update_public_metadata_for_multiple_items(
         variables,
         permissions=[permissions] if permissions else None,
     )
-    response = get_graphql_content(response)
+    response = get_graphql_content(response, ignore_errors=True)
```

Comment:
This change affects that `get_graphql_content` will not check permission in all meta tests.